src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -45,7 +47,7 @@ public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Versi
 
         private void WriteToolInfo(string name, string fileVersion, Version assemblyVersion)
         {
-            _writer.WriteObjectStart("toolInfo");
+            _writer.WriteObjectStart("tool");
             _writer.Write("name", name);
             _writer.Write("version", assemblyVersion.ToString(fieldCount: 3));
```

Comment:
There are two issues:

1) Should there be a toolInfo.loggerVersion property?
2) If there were, would Nick use it here?

If the answer to #2 is "no", we can defer #1 to future.

So... why did you think Nick would use a loggerVersion property? Your scenario was " tools that integrate some other component for producing SARIF ". But Nick isn't using the SARIF SDK; the logging code is built into Roslyn. Why doesn't the version of the Roslyn compiler suffice?

---
In reply to: [62950703](https://github.com/dotnet/roslyn/pull/11233#discussion_r62950703) [](ancestors = 62950703)